main.css

```diff
@@ -1,12 +1,14 @@
 body {
   background: linear-gradient(rgb(255, 166, 0), rgb(23, 168, 204));
+  background-repeat: no-repeat;
   height: 100%;
   text-align: center;
   font-family: 'Rubik Maps';
+  padding: 0px 10px;
 }
 
 #about-britnay {
-  border: white dashed 2px;
+  border: white dashed 3px;
```

Comment:
A way to consolidate this CSS file is to nest your styling. Instead of `#about-britnay` and then whatever tag. You can target the class and then give each HTML element it's specific styling. 

i.e. 
```
#about-britnay {
border: white dashed 2px;
  border: white dashed 3px;
p {
 margin: 0px;
  padding: 0px;
} 

h1 {
margin: 0px;
  padding: 0px;
}
}
```
Click out this: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

Reply:
Okay thank you!